config/initializers/clean_users.rb

```diff
@@ -0,0 +1,7 @@
+lambda do
+  config = Rails.application.config
+  config.after_initialize do
+    interval = Figaro.env.clean_users_job_frequency_seconds || 30.days
+    CleanUsersJob.new.perform(interval.to_i)
+  end
+end.call
```

Comment:
what's the purpose fo doing this as an IIFE? Scoping? Local variables in Ruby are not accessible across files so I think it would be fine to do this without.

And the assigning to `config` is not needed either? This could all be:

```ruby
Rails.application.config.after_initialize do
  interval = Figaro.env.clean_users_job_frequency_seconds || 30.days
  CleanUsersJob.new.perform(interval.to_i)
end
```

Reply:
I had an earlier revision where I thought it made sense, but clearly I didn't clean it up well enough.  Thanks for the pointers.